console/src/assets/i18n/de.json

```diff
@@ -248,13 +248,13 @@
     "SETTINGS": {
       "TITLE": "Einstellungen",
       "GENERAL": "Allgemein",
-      "IDP": "Identity Provider",
+      "IDP": "Identitätsanbieter",
       "SECURITY": "Passwort und Sicherheit",
       "KEYS": "Schlüssel",
-      "PAT": "Personal Access Tokens",
-      "USERGRANTS": "Authorizations",
-      "MEMBERSHIPS": "Memberships",
-      "METADATA": "Metadata"
+      "PAT": "Persönliche Zugriffstoken",
+      "USERGRANTS": "Berechtigungen",
+      "MEMBERSHIPS": "Mitgliedschaften",
```

Comment:
```suggestion
      "MEMBERSHIPS": "Memberships",
```

Reply:
hm, this is a hard one

Reply:
```suggestion
      "MEMBERSHIPS": "Manager-Rollen",
```

This is consistent with the title
![image](https://user-images.githubusercontent.com/12727842/223695344-f7906027-2c19-4aed-be50-f005eb7edb88.png)